cvmfs/sync_item_dummy.h

```diff
@@ -25,6 +25,7 @@ class SyncItemDummyDir : public SyncItemNative {
  public:
   catalog::DirectoryEntryBase CreateBasicCatalogDirent() const;
   SyncItemType GetScratchFiletype() const;
+  virtual void IsPlaceholderDirectory() const { rdonly_type_ = kItemDir; }
```

Comment:
One more thing, can we rename this to `MakePlaceholderDirectory` or similar?  I got for a moment confused and thought this is a boolean function.

Reply:
It seems a quite good idea, I get confused too actually. Fixing it.